Source/Core/Core/PowerPC/JitCommon/JitCache.h

```diff
@@ -17,17 +17,9 @@
 
 class JitBase;
 
-// A JitBlock is block of compiled code which corresponds to the PowerPC
-// code at a given address.
-//
-// The notion of the address of a block is a bit complicated because of the
-// way address translation works, but basically it's the combination of an
-// effective address, the address translation bits in MSR, and the physical
-// address.
-struct JitBlock
+#pragma pack(push, 1)
```

Comment:
Why is packing necessary here? This can also result in more inefficient code being generated. It also makes it way easier for memory alignment violations to occur.

Reply:
okay. I will remove this.